internal_workflow_testsuite.go

```diff
@@ -956,20 +961,20 @@ func (env *testWorkflowEnvironmentImpl) newTestActivityTaskHandler(taskList stri
 	return taskHandler
 }
 
-func newTestActivityTask(workflowID, runID, activityID, activityType string, input []byte) *shared.PollForActivityTaskResponse {
+func newTestActivityTask(workflowID, runID, activityID string, params executeActivityParameters) *shared.PollForActivityTaskResponse {
 	task := &shared.PollForActivityTaskResponse{
 		WorkflowExecution: &shared.WorkflowExecution{
 			WorkflowId: common.StringPtr(workflowID),
 			RunId:      common.StringPtr(runID),
 		},
 		ActivityId:                    common.StringPtr(activityID),
 		TaskToken:                     []byte(activityID), // use activityID as TaskToken so we can map TaskToken in heartbeat calls.
-		ActivityType:                  &shared.ActivityType{Name: common.StringPtr(activityType)},
-		Input:                         input,
+		ActivityType:                  &shared.ActivityType{Name: common.StringPtr(params.ActivityType.Name)},
+		Input:                         params.Input,
 		ScheduledTimestamp:            common.Int64Ptr(time.Now().UnixNano()),
-		ScheduleToCloseTimeoutSeconds: common.Int32Ptr(60),
+		ScheduleToCloseTimeoutSeconds: common.Int32Ptr(params.ScheduleToCloseTimeoutSeconds),
 		StartedTimestamp:              common.Int64Ptr(time.Now().UnixNano()),
-		StartToCloseTimeoutSeconds:    common.Int32Ptr(60),
```

Comment:
not sure I understand. It is not easy to trigger timeout in unit test. It is only possible to trigger if you are debug it and step though which takes long time.